plugins/zns/zns.c

```diff
@@ -223,7 +223,20 @@ static int zone_mgmt_send(int argc, char **argv, struct command *cmd, struct plu
 	if (fd < 0)
 		return errno;
 
-	if (cfg.data_len) {
+	if (!cfg.zsa) {
```

Comment:
May want to limit it to the only supported zone send actions. Then we avoid one trying to issue reserved actions.

Reply:
since Invalid Zone State Transition (device should return this error code) are exist, I think It can be better with not checking reserved actions. And also there is other command for each action. action limit by each command, without limit for send to check the device return right failure.

Reply:
And also if the command extended with other actions for the future, then will always need to be changed. if does that.

Reply:
I am not sure about the precedence around this one. Keith can probably decide.